ai-platform-unified/notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -1,820 +1,1037 @@
 {
```

Comment:
<p>This has some kind of (cut &amp; paste?) formatting glitch-- the top image should not be included, just the 'partially-expanded DAG' image.  It should look like this:</p><p>https://screenshot.googleplex.com/B3tNppYwyFGrt2H.png</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>I don't understand the issue. Not sure what is meant 'the top image should not be included'. This is your existing PNG. Screenshot looks the same to me.</p>